ansible_base/feature_flags/urls.py

```diff
@@ -6,7 +6,7 @@
 app_name = FeatureFlagsConfig.label
 
 api_version_urls = [
-    path('feature_flags_state/', views.FeatureFlagsStateListView.as_view(), name='featureflags-list'),
+    path('feature_flags_state/', views.FeatureFlagsStateListView.as_view(), name='feature-flags-state-list'),
```

Comment:
Shouldn't this be feature_flags_state-list?

Reply:
created a JIRA ticket for this: https://issues.redhat.com/browse/AAP-47290